GoogleUtilities.podspec

```diff
@@ -36,7 +36,10 @@ other Google CocoaPods. They're not intended for direct public usage.
   }
 
   s.subspec 'Environment' do |es|
-    es.source_files = 'GoogleUtilities/Environment/**/*.[mh]'
+    es.source_files = [
+      'GoogleUtilities/Environment/**/*.[mh]',
+      'third_party/IsAppEncrypted/**/*.[mh]'
```

Comment:
Previously when you reviewed @paulb777 , I had `third_party/Environment` as the directory that contains `IsAppEncrypted. My thinking was that this would be all the third party code for the GULsEnvironment target. But thinking about it, it's probably best to be specific as possible and make each subdirectory within third_party map to one component of 3P code.